inlong-audit/audit-sdk/src/main/java/org/apache/inlong/audit/AuditReporterImpl.java

```diff
@@ -502,4 +501,22 @@ private void checkFlushTime() {
             }
         });
     }
+
+    /**
+     * Generate success and failure, real-time and non-real-time, retry, discard and other Audit item IDs through the baseline Audit ID.
+     *
+     * @param baseAuditId
+     * @param success
+     * @param isRealtime
+     * @param discard
+     * @param retry
+     * @return
+     */
+    public int buildAuditId(AuditIdEnum baseAuditId,
+            boolean success,
+            boolean isRealtime,
+            boolean discard,
+            boolean retry) {
+        return AuditManagerUtils.buildAuditId(baseAuditId, success, isRealtime, discard, retry);
+    }
```

Comment:
Why add a method that just calls a public static method `AuditManagerUtils.buildAuditId`?

Reply:
Make sense